CHANGELOG.md

```diff
@@ -35,6 +35,24 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
+## [v2.7.0] - 04-11-2023
+Version 2.7.0 brings many breaking changes with Oracle 2.0.
+
+### State Machine Breaking Changes
+* (x/gov) [\#651](https://github.com/shentufoundation/shentu/pull/651) Extract all customized parts out of the standard gov module.
```

Comment:
How about:
Achieve the compatibility with Cosmos gov interface by removing customized implementation from the gov module.